code/src/App.js

```diff
@@ -1,12 +1,18 @@
-import React from 'react'
-import data from './data.json'
+import React from 'react';
+import data from './data.json';
+import { MusicList } from './MusicList';
 
 console.log(data)
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <main>
+      <header>
```

Comment:
At the end I see you didn't make it with header in the same line as main content of app :(
If you are more curious about that, I suggest you to ask Stack Overflow question and tag me ;)